app/src/layouts/organizationsControl/organizationsControl.jsx

```diff
@@ -48,9 +48,9 @@ export const OrganizationsControl = ({
           {isExtendedNav ? (
             <>
               <i className={cx('arrow-icon')}>{Parser(ArrowLeftIcon)}</i>
-              <Link to={link} className={cx('organization-btn')} onClick={closeSidebar}>
+              <NavLink to={link} className={cx('organization-btn')} onClick={closeSidebar}>
                 <div className={cx('top-title')}>{titles.topTitle}</div>
-              </Link>
+              </NavLink>
```

Comment:
Right now it will be active on all levels.

![image](https://github.com/user-attachments/assets/9ab174c8-1955-4b2f-873a-ac56cd9c11c6)

We've discussed this case with BA previously, so may be leave it as a `Link` here?

Reply:
This is a different element. The link does not work without the NavLink.
![image](https://github.com/user-attachments/assets/12b8fe8c-faf6-4a15-940f-41a02e0cb76c)
It is necessary to create a ticket for the bug.

Reply:
@BlazarQSO ok, got it, contact QA & BA in this case.